.github/workflows/deploy-website.yml

```diff
@@ -36,8 +36,10 @@ jobs:
           working-directory: packages/website
           install: false
           record: true
-          start: yarn serve
-          wait-on: "http://127.0.0.1:9000"
+          start: yarn develop
```

Comment:
Why was the switch needed?

Reply:
because when we build the production bundle without GATSBY_API_URL we rely on the current url to be a portal - when we serve from localhost we error out the upload because localhost is not a portal 

`yarn develop` sets GATSBY_API_URL to siasky.net so running from localhost works, other solution would have to be a have a separate job that for building prod artifact, separate job to build test artifact, and separate job for uploading the prod artifact when tests succeed - seemed a bit much tbh :)

Reply:
Makes sense, thanks for the explanation 🍻